File: templates/Features/DragAndDropVB/Services/DragAndDrop/ListViewDropConfiguration.vb
Namespace Services.DragAndDrop

    Public Class ListViewDropConfiguration
        Inherits DropConfiguration

        Public Shared ReadOnly DragItemsStartingCommandProperty As DependencyProperty = DependencyProperty.Register("DragItemsStartingCommand", GetType(ICommand), GetType(DropConfiguration), New PropertyMetadata(Nothing))

        Public Shared ReadOnly DragItemsCompletedCommandProperty As DependencyProperty = DependencyProperty.Register("DragItemsCompletedCommand", GetType(ICommand), GetType(DropConfiguration), New PropertyMetadata(Nothing))

        Public Property DragItemsStartingCommand As ICommand
            Get
                Return CType(GetValue(DragItemsStartingCommandProperty), ICommand)
            End Get

            Set(value As ICommand)
                SetValue(DragItemsStartingCommandProperty, value)
            End Set
        End Property

        Public Property DragItemsCompletedCommand As ICommand
            Get
                Return CType(GetValue(DragItemsCompletedCommandProperty), ICommand)
            End Get

            Set(value As ICommand)
                SetValue(DragItemsCompletedCommandProperty, value)
            End Set
        End Property
    End Class
End Namespace

File: templates/Uwp/Features/DragAndDrop/Services/DragAndDrop/ListViewDropConfiguration.cs
using System.Windows.Input;
using Windows.UI.Xaml;

namespace Param_RootNamespace.Services.DragAndDrop
{
    public class ListViewDropConfiguration : DropConfiguration
    {
        public static readonly DependencyProperty DragItemsStartingCommandProperty =
            DependencyProperty.Register("DragItemsStartingCommand", typeof(ICommand), typeof(DropConfiguration), new PropertyMetadata(null));

        public static readonly DependencyProperty DragItemsCompletedCommandProperty =
            DependencyProperty.Register("DragItemsCompletedCommand", typeof(ICommand), typeof(DropConfiguration), new PropertyMetadata(null));

        public ICommand DragItemsStartingCommand
        {
            get { return (ICommand)GetValue(DragItemsStartingCommandProperty); }
            set { SetValue(DragItemsStartingCommandProperty, value); }
        }

        public ICommand DragItemsCompletedCommand
        {
            get { return (ICommand)GetValue(DragItemsCompletedCommandProperty); }
            set { SetValue(DragItemsCompletedCommandProperty, value); }
        }
    }
}
